docs/docs/how-tos/review-tool-calls-functional.ipynb

```diff
@@ -0,0 +1,585 @@
+{
```

Comment:
should we lift a short example of the tool review flow to the top? same as in the non-functional how to?